### ISSUE TITLE
Refactoring of questions service

### ISSUE THREAD

**Issue by ChristianFN2** [Arquisoft/wiq_en1a#76]
Currently the question service has only the functionallity for the flag questions and it is not extensible, so it should be refactored to be able to easily add more questions in the future.

**Comment by ChristianFN2** [comment]
I think there is an inherent problem in how I constructed the query (if I want it to be generic), so I will reconstruct it and parametrize the method to accept several cathegories of image questions. For that I will use this official tutorial https://www.wikidata.org/wiki/Wikidata:SPARQL_tutorial, and this service to check the functioning of the queries and construct them easier https://query.wikidata.org/ (I had used it before and it is in the source links #4 ).

When I have designed the queries and the method, I expect the methodology of adding a new cathegory (for image questions) to be the following:
1. We think in two related concepts that comply with the structure of the designed method.
2. We use the query service to check if the result is the expected in quality and quantity
3. We add a new endpoint to the service using the parametrized method (No need to create a new method)
